model_zoo/census_model_sqlflow/dnn/census_feature_column.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
+
+CATEGORICAL_FEATURE_KEYS = [
+    "workclass",
+    "education",
+    "marital-status",
+    "occupation",
+    "relationship",
+    "race",
+    "sex",
+    "native-country",
+]
+NUMERIC_FEATURE_KEYS = [
+    "age",
+    "capital-gain",
+    "capital-loss",
+    "hours-per-week",
+]
+LABEL_KEY = "label"
+
+
+def get_feature_columns():
+    feature_columns = []
+
+    for numeric_feature_key in NUMERIC_FEATURE_KEYS:
+        numeric_feature = tf.feature_column.numeric_column(numeric_feature_key)
+        feature_columns.append(numeric_feature)
+
+    for categorical_feature_key in CATEGORICAL_FEATURE_KEYS:
+        embedding_feature = tf.feature_column.embedding_column(
```

Comment:
Will these be automatically generated by the SQL statement?

Reply:
Yes, the python file is the corresponding generated code from SQL statement.

Reply:
Got it. Sounds good to me then.